src/combinator/mod.rs

```diff
@@ -82,7 +82,7 @@
 //! ## Remaining combinators
 //!
 //! - [`empty`]: Returns a value without consuming any input, always succeeds
-//! - [`fail`]: Inversion of `success`. Always fails.
+//! - [`fail`]: Inversion of `empty`. Always fails.
```

Comment:
```suggestion
//! - [`fail`]: Inversion of [`empty`]. Always fails.
```
Help catch problems like this in the future :)